libs/sr/formula-ui/src/char/tagFieldMap.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  compileTagMapValues,
+  TagMapSubset,
+} from '@genshin-optimizer/pando/engine'
+import type { TagField } from '@genshin-optimizer/pando/ui-sheet'
+import { isTagField } from '@genshin-optimizer/pando/ui-sheet'
+import type { Tag } from '@genshin-optimizer/sr/formula'
+import { keys } from '@genshin-optimizer/sr/formula'
+import { uiSheets } from './sheets'
+import { charBaseUiSheet } from './sheets/CharBase'
+
+const tagValue: Array<{ tag: Tag; value: TagField }> = []
+Object.values(uiSheets).forEach((sheet) => {
```

Comment:
Should we be crawling *another* sheet to add to `tagValue`? Can we somehow define both the sheets and `tagValue` simultaneously? If not, this looks fine enough.

Reply:
i doubt the shape of sheets will change, so this crawling function will rarely need to be updated. I rather do this than to have to export a separate thing in sheets that will be mostly boilerplate per sheet.